tests/functional/aws-node-sdk/test/object/websiteGet.js

```diff
@@ -576,5 +577,230 @@ describe('User visits bucket website endpoint', () => {
                 });
             });
         });
+
+        describe('with key x-amz-website-redirect-location header ' +
```

Comment:
Let's create a separate test file called websiteRuleMixing and put the new tests in there so we are not mixing promises and callbacks in the same file.